extensions/wikia/Listusers/SpecialListusers_helper.php

```diff
@@ -89,41 +86,36 @@ function setOrder    	( $orders = array() ) {
 				}
 			}
 		}
+
+		// SUS-3207 - order by number of edits (descending) by default
 		if ( empty( $this->mOrder ) ) {
-			$this->mOrder[] = 'user_name ASC';
+			$this->mOrder[] = 'edits DESC';
 		}
 	}
 
-	function getFilterGroup () { return $this->mFilterGroup; }
 	function getGroups   	() { return $this->mGroups; }
-	function getUserName	() { return $this->mUserName; }
-	function getEdits    	() { return $this->mEdits; }
-	function getLimit    	() { return $this->mLimit; }
-	function getOffset   	() { return $this->mOffset; }
-	function getOrder    	() { return $this->mOrder; }
 
 	public function loadData() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this function "loadData" is 26 which is greater than 20 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)

[project-id]: wikia:app
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This function "loadData" has 200 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)

[project-id]: wikia:app